packages/agent/app/clear-persistent-storage.ts

```diff
@@ -0,0 +1,34 @@
+import { getIndexedDBConfig } from './indexed-db-config';
+
+export function* clearPersistentStorage() {
+  localStorage.clear();
+  sessionStorage.clear();
+
+  yield clearIndexedDB();
+}
+
+function* clearIndexedDB() {
+  let { openedDBNames } = getIndexedDBConfig();
+
+  // delete any database that were opened in previous lanes
+  for (let dbName of openedDBNames) {
+    let request: IDBRequest = window.indexedDB.deleteDatabase(dbName);
+    yield idbRequest(request);
```

Comment:
This ends up being serial, I think? Probably not really worth it to parallelize this, right?

Reply:
Hmm... deletes are pretty fast, but then again, why not make it parallel? My main reason for not wanting to do parallel was not having to use `fork()` which is quasi-deprecated, but we don't have a great replacement.

Reply:
I went ahead and parallelized. There's no reason we should expose ourselves to a performance hit when it really makes no difference either way. The goal of effection after all is to make parallel so cheap and risk free that these decisions are a no-brainer, is it not?